hw/dv/data/dsim/dsim.hjson

```diff
@@ -0,0 +1,119 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "{job_prefix} {DSIM_HOME}/dsim"
+  run_cmd:    "{job_prefix} {DSIM_HOME}/dsim"
+
```

Comment:
Thanks for adding this.
just tried to run dsim with this PR. Perhaps add below as our flow expects tool_srcs to be set. With that, dsim flow looks good, but need to update some SV codes to pass compilation (we can do that later), as dsim has different requirement on some syntax.
```
  // Indicate the tool specific helper sources - these are copied over to the	
  // {tool_srcs_dir} before running the simulation.
  // TODO, there is no dsim tool file, point to vcs for now to avoid error from script
  tool_srcs:  ["{proj_root}/hw/dv/tools/vcs/*"]
```

Reply:
Done